tests/Resources/Page/LoginPage.robot

```diff
@@ -19,6 +19,14 @@ Is OpenShift OAuth Login Prompt Visible
    ${result} =  Evaluate  ${login_prompt_visible} and ${oauth_login}
    [Return]  ${result}
 
+OpenShift Login Visible While Reaching RHODS
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Page Should Contain     Log in with
```

Comment:
lines 24 to 28 are already implemented in keyword `Is OpenShift Login Visible` , pls look if you can use it directly

Reply:
Yes I have modified and now used it directly.